client/components/screen-options-tab/screen-switcher.js

```diff
@@ -3,25 +3,48 @@
  */
 import React from 'react';
 import { useI18n } from '@wordpress/react-i18n';
+import { useSelector } from 'react-redux';
+
+/**
+ * Internal dependencies
+ */
+import { getSelectedSiteId } from 'calypso/state/ui/selectors';
+import { getSiteAdminUrl } from 'calypso/state/sites/selectors';
+import { addQueryArgs } from 'calypso/lib/route';
 
 /**
  * Style dependencies
  */
 import './style.scss';
 
-const ScreenSwitcher = () => {
+const ScreenSwitcher = ( { onSwitch, wpAdminPath } ) => {
 	const { __ } = useI18n();
 
+	const siteId = useSelector( getSelectedSiteId );
+	let fullWpAdminUrl = useSelector( ( state ) => getSiteAdminUrl( state, siteId, wpAdminPath ) );
+
+	// We indicate that the WP Admin view is preferred with the `preferred-view` query param. WP Admin will
+	// use that param to store the preference, so next times the user visits the page via the sidebar menu
+	// it will default to the WP Admin page.
+	fullWpAdminUrl = addQueryArgs( { 'preferred-view': 'classic' }, fullWpAdminUrl );
```

Comment:
Can we create a new `const` instead of mutating the existing `let` here (and change the `let` to `const`?

`const fullWpAdminUrlWithQueryArgs = addQueryArgs( { 'preferred-view': 'classic' }, fullWpAdminUrl );`

Reply:
What would be the benefit of doing that? I always try to refrain from adding more variables than necessary because that increases the cognitive load needed for reading the code. For instance, in this case you'd need to remember there are 2 variables and what are the differences between them.

Reply:
Maybe it's just preference but I see the cognitive load as less. Whenever I see a `let` I have to be conscious of the fact that it's value (and perhaps type) may change further down the code.

Making the values immutable gives me more comfort and an opportunity to be more descriptive in their declarations when naming them.

Reply:
https://github.com/Automattic/wp-calypso/pull/53866/commits/17de070e145e50997625cb39a0036eecc5978e0d